Updated asyncio code snippet (#7662)
getsentry/sentry-docs: src/platforms/python/common/configuration/integrations/asyncio.mdx

## Configure

> Add `AsyncioIntegration()` to your `integrations` list. Make sure that you call `sentry_sdk.init()` at the beginning of your async loop:

Reviewer: ```suggestion
Add `AsyncioIntegration()` to your list of `integrations` and be sure to call `sentry_sdk.init()` at the beginning of your async loop:
```
